engine/swarm/index.md

```diff
@@ -3,34 +3,46 @@ description: Docker Engine swarm mode overview
 keywords: docker, container, cluster, swarm
 title: Swarm mode overview
 redirect_from:
-- /swarm/overview/
+- /api/swarm-api/
+- /engine/userguide/networking/overlay-standalone-swarm/
+- /network/overlay-standalone.swarm/
+- /release-notes/docker-swarm/
+- /swarm/
```

Comment:
One consideration could be to redirect these to https://docs.docker.com/engine/ instead of https://docs.docker.com/engine/swarm/ I thought this would be more on-topic, but open to suggestions

Reply:
Yup, I think it makes sense to redirect to https://docs.docker.com/engine/swarm/.